djl-spring-boot-mxnet-autoconfigure/src/main/java/ai/djl/spring/configuration/DjlConfigurationProperties.java

```diff
@@ -19,31 +19,73 @@
 @ConfigurationProperties("djl")
 public class DjlConfigurationProperties {
 
+    private ApplicationType applicationType;
+    private Class<?> inputClass;
+    private Class<?> outputClass;
+    private EngineType engineType;
+    private String artifactId;
+    private Map<String, Object> arguments;
```

Comment:
We need test coverage on these. While DJL testing (through spring boot starter) is not the goal of the project, testing configurations is needed. Apart from just standard engineering process, it is a requirement from Pivotal since we intend to pass engineering review from Pivotal to be included in http://start.spring.io.

Reply:
Added test for new properties.